src/convertToObject.js

```diff
@@ -6,7 +6,27 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssPropertiesDotComa = sourceString
+    .split(';')
+    .map((item) => item.trim())
+    .filter((item) => item.length > 0);
```

Comment:
[CODE KNOWLEDGE] - There is no need to make a copy of the array before applying the `map` and `filter` methods, as these methods do not mutate the original array. You can chain these methods directly after the `split`.